source/game/field/obj/ObjectBreakable.hh

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "game/field/obj/ObjectCollidable.hh"
+
+namespace Field {
+
+class ObjectBreakable : public ObjectCollidable {
+public:
+    ObjectBreakable(const System::MapdataGeoObj &params) : ObjectCollidable(params) {}
+    ~ObjectBreakable() = default;
+
+    /// @addr{0x8076EC68}
+    void init() override {
+        enableCollision();
+    }
```

Comment:
Does it still sync if you get rid of this? I assume the `Intangible` bit is not set automatically but I could be wrong. If we don't need this, take it out too